.github/workflows/main.yml

```diff
@@ -12,21 +13,25 @@ jobs:
       - name: 🚚 Get latest code
         uses: actions/checkout@v3
 
-      - name: 👷‍♂️ Setup Node.js
-        uses: actions/setup-node@v3
-        with:
-          node-version: 18
-
       - name: 📦 Setup pnpm
         uses: pnpm/action-setup@v2
         with:
           version: 7.25.0
 
+      - name: 👷‍♂️ Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'pnpm'
+
       - name: 💿 Install dependencies
         run: pnpm install
 
       - name: 🛠️ Build app
-        run: npm run build --prod
+        run: pnpm build --prod
+
+      - name: 🔍 Lint app
```

Comment:
Hi 👋,

Den Lint-Step brauchen wir nicht, weil der im Build-Step bereits gemacht wird.

<img width="653" alt="CleanShot 2023-05-13 at 08 03 02@2x" src="https://github.com/co-IT/homepage/assets/444278/289d9042-fc5c-4929-981f-5cf13c31ded4">